### DISCUSSION TITLE
不换的小窝

### DISCUSSION THREAD

**Discussion by iambinlin** [YunYouJun/hexo-theme-starry#280] [Show 分享展示]
name: 不换的小窝
url: https://dev.bigdreamer.cc
avatar:  http://1.gravatar.com/userimage/220011667/8f187602081d27b42c2d08cb1782268d
desc:  人终归要跟自己的平凡和解

**Comment by YunYouJun** [comment]
可以参考此处 https://github.com/YunYouJun/hexo-theme-yun/discussions/97 ，使用自助示例站点。
